2_Operations/apps.md

```diff
@@ -40,9 +40,31 @@ Slack can be a [big distraction](https://m.signalvnoise.com/is-group-chat-making
 
 Signal is to be used primarily for sending confidential information that must never be shared over Slack, and timely instant messaging at conferences and IRLs. All other instant messaging should remain on Slack.
 
-## OpenVPN
+## VPN
+
+We use NordVPN to securely tunnel Internet traffic when we are not on our home network.
+
+#### Usage
+
+1. Go to https://nordvpn.com and install their app. It works both on desktop and mobile devices.
```

Comment:
Should add a note that on macOS, app store version does not have Dedicated IPs section ?
https://niteo.slack.com/archives/C071TQ6LE/p1566301274017700